packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -333,13 +334,15 @@ const SettingsSidebarContainer = ({
                         </Collapsible>
                       );
                   })}
-                <VerticalTabItem
-                  name={t("add_a_team")}
-                  href={`${WEBAPP_URL}/settings/teams/new`}
-                  textClassNames="px-3 items-center mt-2 text-emphasis font-medium text-sm"
-                  icon={Plus}
-                  disableChevron
-                />
+                {!user?.organizationId || user?.organization.isOrgAdmin ? (
+                  <VerticalTabItem
+                    name={t("add_a_team")}
+                    href={`${WEBAPP_URL}/settings/teams/new`}
+                    textClassNames="px-3 items-center mt-2 text-emphasis font-medium text-sm"
+                    icon={Plus}
+                    disableChevron
+                  />
+                ) : null}
```

Comment:
```suggestion
                {!user?.organizationId || user?.organization.isOrgAdmin && (
                  <VerticalTabItem
                    name={t("add_a_team")}
                    href={`${WEBAPP_URL}/settings/teams/new`}
                    textClassNames="px-3 items-center mt-2 text-emphasis font-medium text-sm"
                    icon={Plus}
                    disableChevron
                  />
                )}
```
Wouldn't this be better?